internal/security/secretstore/init.go

```diff
@@ -421,73 +421,76 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 		return false
 	}
 
-	// for mqtt-bus creds
-	var mqttBusCredentials UserPasswordPair
-	if configuration.SecureMessageBus.Type == mqttSecretName {
-		mqttBusCredentials, err = getCredential("security-bootstrapper-mqtt", secretStore, mqttSecretName)
+	// for secure message bus creds
+	var msgBusCredentials UserPasswordPair
+	if configuration.SecureMessageBus.Type != redisSecureMessageBusType &&
+		configuration.SecureMessageBus.Type != noneSecureMessageBusType &&
+		configuration.SecureMessageBus.Type != blankSecureMessageBusType {
+		msgBusCredentials, err = getCredential(internal.BootstrapMessageBusServiceKey, secretStore, messagebusSecretName)
 		if err != nil {
 			if err != errNotFound {
-				lc.Error("failed to determine if MQTT credentials already exist or not: %w", err)
+				lc.Errorf("failed to determine if %s credentials already exist or not: %w", configuration.SecureMessageBus.Type, err)
 				return false
 			}
 
-			lc.Info("Generating new password for MQTT bus")
-			mqttPassword, err := secretStore.GeneratePassword(ctx)
+			lc.Infof("Generating new password for %s bus", configuration.SecureMessageBus.Type)
+			msgBusPassword, err := secretStore.GeneratePassword(ctx)
 			if err != nil {
-				lc.Error("failed to generate password for mqtt-bus")
+				lc.Errorf("failed to generate password for %s", messagebusSecretName)
```

Comment:
Should be consistent with info message above, i.e. type vs secretname. IMO it should be as follows
```suggestion
				lc.Errorf("failed to generate password for %s bus", configuration.SecureMessageBus.Type)
```